app/src/main/java/com/github/houseorganizer/houseorganizer/SettingsActivity.java

```diff
@@ -24,10 +36,42 @@ protected void onCreate(Bundle savedInstanceState) {
         }
     }
 
-    public static class SettingsFragment extends PreferenceFragmentCompat {
+    public static class SettingsFragment extends PreferenceFragmentCompat implements SharedPreferences.OnSharedPreferenceChangeListener {
+
+        private FirebaseFirestore db;
+        private FirebaseUser Uid;
```

Comment:
isnt it simpler to store the mail here ? or even not save anything in a variable since the uid/mail is used once in this class

Reply:
Yes, done